### ISSUE TITLE
Improve online list 

### ISSUE THREAD

**Issue by zachferland** [3box/3box-chatbox#29]
Details pulled from here - https://github.com/3box/3box-js/issues/703

There is some things that can be done in the chatbox to still improve the experience for online ids/profiles. Now with auth later, users are not included in online list until they send a message. We may want to still auth earlier if you know the user wont have to sign a message. Second you probably want to represent num people/peers in room (not just ones who are on line and sent messages, authed at this point now, so doesnt send user join message till then), if not all are authed we could have a total number of peers, then the listmembers still with addresses/name and number of anonymous peers

If want to show number of total online peers, including anonymous, then we can add a simple helper function to 3box-js as well

cc @oznekenzo @michaelsena

**Comment by oznekenzo** [comment]
Yeah, it would be better to show a list of people that are online, not just those that have sent a message. But up to you and Sena whether the time spent enabling that is worth it.
